packages/core/src/RenderingEngine/helpers/createVolumeActor.ts

```diff
@@ -48,45 +49,40 @@ async function createVolumeActor(
   const volumeActor = vtkVolume.newInstance();
   volumeActor.setMapper(volumeMapper);
 
-  const voiRange = volumeActor
-    .getProperty()
-    .getRGBTransferFunction(0)
-    .getRange()
-    .slice();
+  // If the volume is composed of imageIds, we can apply a default VOI based
```

Comment:
Might be good to write an example of when there might not be imageIds, as this won't be obvious to new users.